### PR TITLE
bugfix - resolve issue #1155

### LINKED ISSUE #1155: Hoverfly templating issue with number greater than 9007199254740991
## Description of the bug
I am trying  to get data from request to the response via templating.While trying it is observed for numbers greater than 9007199254740991 in the request , the response gets changed .
**Example** : 
If I am passing this as the request body :
"Input": {
        "inputNo1": "100",
        **"inputNo2": "0009007199254740999",**
        "inputNo3": "001"
    }

**Output Response:** 
"Output": {
        "outputNo1": "100",
        **"outputNo2": "0009007199254741000",**
        "outputNo3": "1"
}

**Template used : "{{ Request.Body 'jsonpath' '$.Input.inputNo2' }}"**
## Steps to reproduce the issue
Use the above template to extract the data from request .
Here the data is send as string and the template is also placed within "".
### Observed result
**Hoverfly error messages seen (If none, say none)**
``` None
(paste here)
```
**If possible, add screenshots to help explain your problem**

## Additional relevant information
1. Hoverfly version:  
     | hoverctl | master-4270 |
     | hoverfly | v1.9.5 
2. Anything that might help us to diagnose the problem: Could it be due to floating-point precision limitations? What is the solution to it .

### PR DESCRIPTION
**Pull request by kapishmalik**
Resolves #1155

### CODE REVIEW

File: core/util/util.go [hunk 1]
```diff
@@ -399,14 +399,14 @@ func jsonPath(query, toMatch string) interface{} {
...
-		result = strconv.Itoa(intResult)
-	}
+	//// Jsonpath library converts large int into a string with scientific notion, the following
+	//// reverts that process to avoid mismatching when using the jsonpath result for csv data lookup
+	//floatResult, err := strconv.ParseFloat(result, 64)
+	//// if the string is a float and a whole number
+	//if err == nil && floatResult == float64(int64(floatResult)) {
+	//	intResult := int(floatResult)
+	//	result = strconv.Itoa(intResult)
+	//}
```
Comment (tommysitu) [inline core/util/util.go:409]: but this introduces another issue...

Comment (kapishmalik) [reply core/util/util.go:409]: This is not correct way to do things. I am not sure why we added this hack.

Comment (tommysitu) [reply core/util/util.go:409]: so what would be the correct way?

Comment (kapishmalik) [reply core/util/util.go:409]: @tommysitu  is that change still needed?

Comment (tommysitu) [reply core/util/util.go:409]: yes, it's needed. But commenting it out, several tests will fail, eg. `Test_ApplyTemplate_PutAndGetValue`

Comment (kapishmalik) [reply core/util/util.go]: @tommysitu I have added handling for it.

**Review by tommysitu — APPROVED** [review]
seems like a decent solution, thanks!

**Pull request merged**